plugins/node/opentelemetry-instrumentation-mongodb/package.json

```diff
@@ -7,8 +7,9 @@
   "repository": "open-telemetry/opentelemetry-js-contrib",
   "scripts": {
     "docker:start": "docker run -e MONGODB_DB=opentelemetry-tests -e MONGODB_PORT=27017 -e MONGODB_HOST=127.0.0.1 -p 27017:27017 --rm mongo",
-    "test": "nyc ts-mocha -p tsconfig.json --require '@opentelemetry/contrib-test-utils' 'test/**/mongodb-v3.test.ts'",
-    "test-new-versions": "nyc ts-mocha -p tsconfig.json --require '@opentelemetry/contrib-test-utils' 'test/mongodb-v4**.test.ts'",
+    "test-v3": "nyc ts-mocha -p tsconfig.json --require '@opentelemetry/contrib-test-utils' 'test/**/mongodb-v3.test.ts'",
```

Comment:
I think this script should be kept as `test` as that is the one that will be executed by default. :slightly_smiling_face:

Reply:
Ah yeah makes sense. By default should it run only the test for `v3`, or should it run all versions ? 👀

Reply:
By default, we had it run only for v3, so I think keeping just `v3` is fine. The test-all-versions script will take care of running the other tests (and that will run if the `pkg:instrumentation-mongodb` label is applied to the PR) :slightly_smiling_face: